lib/monitor/watch.js

```diff
@@ -53,6 +53,7 @@ function watch() {
         ignored: ignored,
         persistent: true,
         usePolling: config.options.legacyWatch || false,
+        interval: config.options.pollingInterval,
```

Comment:
@remy chokidar falls back to it's default (100 ms) if interval is undefined. I can explicitly omit the option when undefined if you'd prefer that.